src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -201,6 +201,98 @@ After a short (less than 5 minute) delay, the status will change from **Pending
 
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+## Step 5. (Optional) Configure private endpoint trusted owners
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/limited-access.md %}
+{{site.data.alerts.end}}
+
+Optionally, you can restrict the AWS accounts that can connect to your cluster privately. During Limited Access, to configure trusted owners, you must use the [CockroachDB CockroachDB {{ site.data.products.cloud }} API]({% link cockroachcloud/cloud-api.md %}) or [Terraform Provider](https://github.com/cockroachdb/terraform-provider-cockroach/).
+
+Keep the following in mind:
+- This feature can be enabled only on clusters created after your organization is enrolled in the Limited Access.
+- After your organization is enrolled, an attempt to create a PrivateLink connection from another AWS account will fail.
+- After your organization is enrolled, the feature cannot be disabled.
+
+Your service account must have one of the following roles on the cluster, either directly or by inheritance:
+
+- [Cluster Administrator]({% link cockroachcloud/authorization.md %}#cluster-administrator) to add or remove private endpoint trusted owners.
+- [Cluster Operator]({% link cockroachcloud/authorization.md %}#cluster-operator) to list or get details about private endpoint trusted owners.
+
+### Add a private endpoint trusted owner
+
+To [add a private endpoint trusted owner](https://www.cockroachlabs.com/docs/api/cloud/v1#post-/api/v1/clusters/-cluster_id-/networking/private-endpoint-trusted-owners):
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request POST \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/networking/private-endpoint-trusted-owners \
+  --header 'Authorization: Bearer {bearer_token' \
+  --header 'content-type: application/json' \
+  --data '{"external_owner_id":"{aws_account_id}","type":"AWS_ACCOUNT_ID"}'
+~~~
+
+Replace:
+- `{cluster_id}`: The ID of the cluster.
+- `{bearer_token}`: The service account's API key.
+- `{aws_account_id}`: The ID of the AWS account to trust.
+
+Next, you can [create an AWS endpoint](#step-2-create-an-aws-endpoint).
+
+### List private endpoint trusted owners
+
+To [list private endpoint trusted owners](https://www.cockroachlabs.com/docs/api/cloud/v1#get-/api/v1/clusters/-cluster_id-/networking/private-endpoint-trusted-owners) for a cluster:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request GET \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/networking/private-endpoint-trusted-owners \
+  --header 'Authorization: Bearer {bearer_token}'
+~~~
+
+Replace:
+
+- `{cluster_id}`: The ID of the cluster.
+- `{bearer_token}`: The service account's API key.
+
+### Get details about a trusted owner
+
+To [get details about a private endpoint trusted owner](https://www.cockroachlabs.com/docs/api/cloud/v1#get-/api/v1/clusters/-cluster_id-/networking/private-endpoint-trusted-owners/-owner_id-):
```

Comment:
I think in both this single GET and DELETE requests, we should specify that the owner_id is the CC generated owner_id which is created when a trusted owner is first added. It can be looked up by using the list GET request.